addon/components/variable-number/number.ts

```diff
@@ -68,6 +70,15 @@ export default class VariableNumberPluginNumberComponent extends Component<Args>
     return this.node.attrs[MAXIMUM_VALUE_PNODE_KEY] as number;
   }
 
+  get canConvertToNumber() {
+    try {
+      n2words(Number(this.node.attrs.value), { lang: 'nl' });
+      return true;
+    } catch (e) {
+      return false;
+    }
+  }
```

Comment:
I feel like this would fit together in the file with `numberToWords`. That way you can avoid using n2words at all in this file, making it so (in the future), devs will more quickly grab the safer numbertoWords, instead of using n2words because they see it here. And they just kinda fit together: both are custom functions to handle that n2words can error.